src/test/java/net/datafaker/providers/base/DateAndTimeTest.java

```diff
@@ -262,7 +260,12 @@ void durationTestUnitAsString_equalMinAndMax() {
     void durationTest(long maxValue, String unit) {
         Duration generated = faker.date().duration(maxValue, unit);
         Duration max = Duration.of(maxValue, DateAndTime.str2durationUnit(unit));
-        assertThat(max.compareTo(generated) > 0 || maxValue == 0).as("Duration must be lower than max value").isTrue();
+        if (maxValue == 0) {
+            assertThat(generated).isEqualTo(Duration.ofMillis(0));
+        }
+        else {
```

Comment:
Same line?

Did you run spotless:apply?

Reply:
@kingthorin Fixed.
(but `mvn spotless:apply` doesn't say anything about these lines)

Reply:
thanks, weird